### PR TITLE
Clean docstrings

### PR DESCRIPTION
**Pull request by scovitz**

### CODE REVIEW

File: bond/bond.py [hunk 1]
```diff
@@ -102,6 +102,23 @@ def datalad_undo_last_commit(self):
         reset_proc.check_returncode()
 
     def apply_csv_changes(self, og_prefix, new_prefix):
+        """Applies changes documented in the edited _summary csv
+        and generates the new csv files..
```
Comment (mattcieslak) [inline bond/bond.py:106]: 2 periods?

File: bond/bond.py [hunk 2]
```diff
@@ -102,6 +102,23 @@ def datalad_undo_last_commit(self):
...
+
+        This function looks at the RenameKeyGroup and MergeInto
+        columns and modifies the bids datset according to the
+        specified changs.
+
+        Parameters:
+        -----------
+            og_prefix : str
```
Comment (mattcieslak) [inline bond/bond.py:114]: what does og stand for? maybe worth spelling out

File: bond/bond.py [hunk 3]
```diff
@@ -102,6 +102,23 @@ def datalad_undo_last_commit(self):
...
+                Path prefix to the directory that contains the
+                original _summary and _files csv files
+            new_prefix : str
```
Comment (mattcieslak) [inline bond/bond.py:117]: Is this a path to a directory or a file stem? We confused ourselves with this a lot

File: bond/bond.py [hunk 4]
```diff
@@ -235,12 +265,9 @@ def change_filename(self, filepath, entities):
         else:
             print("FOUND IRREGULAR NUMBER OF JSONS")
 
-    def fieldmaps_ok(self):
-        pass
-
     def _cache_fieldmaps(self):
-        """Searches all fieldmaps and creates a lookup for each file.
-        """
+        '''Searches all fieldmaps and creates a lookup for each file.'''
```
Comment (mattcieslak) [inline bond/bond.py:269]: This function returns a list of misfits. Does that get used elsewhere? if we want to keep it, we should add it to the docstring or we should remove it. A function called _cache suggests that the returned object should be some sort of cache or None

File: bond/bond.py [hunk 5]
```diff
@@ -267,19 +294,39 @@ def _cache_fieldmaps(self):
         return misfits
 
     def get_param_groups_from_key_group(self, key_group):
+        """Splits key groups into param groups based on json metadata.
+
+        Parameters:
+        -----------
+            og_prefix : str
```
Comment (mattcieslak) [inline bond/bond.py:301]: same question about og

File: bond/bond.py [hunk 6]
```diff
@@ -401,7 +442,8 @@ def get_all_metadata_fields(self):
         return sorted(found_fields)
 
     def remove_metadata_fields(self, fields_to_remove):
-        """Removes specific fields from all metadata files."""
+        '''Removes specific fields from all metadata files.'''
```
Comment (mattcieslak) [inline bond/bond.py:445]: Why the switches to single quotes? I think double quotes are more common for docstrings

File: bond/bond.py [hunk 7]
```diff
@@ -536,44 +584,11 @@ def _get_param_groups(files, layout, fieldmap_lookup, key_group_name):
 
 
 def _order_columns(df):
+    '''Organizes columns of the summary and files DataFrames.'''
```
Comment (mattcieslak) [inline bond/bond.py:587]: Organizes how?

**Review by mattcieslak — CHANGES_REQUESTED** [review]
Looks good! A few questions

File: bond/bond.py (hunk 4 above)
Comment (scovitz) [reply bond/bond.py:269]: I think the misfits are something Ted wanted to keep track of, so I think we should keep the list as a return. I will update the docstring.

**Review by mattcieslak — APPROVED** [review]

**Pull request merged**